spaceship/lib/spaceship/client.rb

```diff
@@ -491,13 +491,17 @@ def send_shared_login_request(user, password)
         # Unauthorized
         raise InvalidUserCredentialsError.new, "Invalid username and password combination. Used '#{user}' as the username."
       when 403
+        error_string = "Login worked, but access was forbidden. Try logging in on https://appstoreconnect.apple.com and see if any issues are raised to you there."
         # Forbidden
-        if response.body["serviceErrors"][0]["code"] == "-20209"
-          # Apple ID got locked by Apple for security reasons
-          raise InvalidUserCredentialsError.new, "This Apple ID has been locked for security reasons. Visit iForgot to reset your account (https://iforgot.apple.com)."
-        else
-          raise InvalidUserCredentialsError.new, "Login worked, but Apple reported that something was wrong anyway. Historically this has happened when an Apple ID got lost on Apple's end. See https://github.com/fastlane/fastlane/issues/14398"
+        case response.body["serviceErrors"][0]["code"]
+        when "-20209"
```

Comment:
Can we add a comment with the full response body here (and for all other error cases) for reference? 
(Or as fixtures for a corresponding test?)